### DISCUSSION TITLE
在umi项目config目录下的文件中怎么使用webpack的require.context?

### DISCUSSION THREAD

**Discussion by wh1246935943** [umijs/umi#7754] [Q&A]
我的项目模块比较多，每个模块又涉及很多的路由，所以给每个模块创建了一个ts文件用来配置路由，然后将所有模块内容通过require.context拿到再配置到config.ts中，但是这样报错了：require.context is not a function;代码如下：
![image](https://user-images.githubusercontent.com/26916703/148167501-6e1fdd85-033d-42db-b025-3b038e0077fb.png)

请问该怎么在config目录下的文件中使用require.context这个api，因为在其他地方是正常的。

**Comment by wangyuan0108** [comment]
这个问题解决了吗？求方法

**Reply by wangyuan0108** [reply]
> 我的项目模块比较多，每个模块又涉及很多的路由，所以给每个模块创建了一个ts文件用来配置路由，然后将所有模块内容通过require.context拿到再配置到config.ts中，但是这样报错了：require.context is not a function;代码如下： ![image](https://user-images.githubusercontent.com/26916703/148167501-6e1fdd85-033d-42db-b025-3b038e0077fb.png)
> 
> 请问该怎么在config目录下的文件中使用require.context这个api，因为在其他地方是正常的。

我也遇到了这个问题

**Reply by fz6m** [reply]
在配置文件里不支持 webpack 语法，一个个导入然后解构吧，不会写多少行的。
